### DISCUSSION TITLE
Shouldn't ripgrep have a documentation site (perhaps with mdbook)?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [BurntSushi/ripgrep#2616] [General] [answered]
Usually for CLIs and projects like rg, the first way I think of looking up all the possible arguments is by googling something like "ripgrep documentation" and expect to find a full comprehensive documentation page or site, sometimes made with mdbook. I actually used GPT 4 to generate a working command searched the argument `--max-count` and didn't find it in the guide. I did find that you can do `rg -h` to see all the possible commands, and I found it there.

**Comment by BurntSushi** [comment, marked as answer]
Have you looked at `rg --help`? Or the markdown files linked from the README? If not, why not?

I'm not inclined to maintain an entirely separate set of docs from what already exists. It seems like a lot of work for little gain.
